src/app.ts

```diff
@@ -0,0 +1,8 @@
+'use strict'
```

Comment:
Think the `'use-strict'`  is redundant here as the tsconfig option `strict: true` enables `alwaysStrict` (https://www.typescriptlang.org/tsconfig/#alwaysStrict) which emits JS in strict mode (now the word strict doesn't look real to me 😆)